Vehicle.h

```diff
@@ -2,28 +2,24 @@
 
 #include <string>
 #include <iostream>
+#include "Highway.h"
 
+struct HighwayPatrol;
 struct Vehicle
 {
-    Vehicle(const std::string& n) : name(n) { }
+    Vehicle(const std::string& n);
     
     //Special member Functions.  See instruction 9) and note in main()
     virtual ~Vehicle() = default;
     Vehicle(const Vehicle&) = default;
     Vehicle& operator=(const Vehicle&) = default;
```

Comment:
v-table warnings can be resolved by eliminating header file implementations.

`= default` is an implementation.
Move these implementations to the cpp file
- [ ] revise